aten/src/ATen/native/quantized/cpu/OnednnUtils.h

```diff
@@ -418,6 +420,7 @@ inline bool is_weight_symmetric_quant(
   return is_symmetric;
 }
 
+#ifdef USE_FBGEMM
```

Comment:
ditto

Reply:
See the above comment. The alternative is I can instead guard the implementation by arch and just return `true` in those cases.

Reply:
FYI, I'm not sure if the use of cpuinfo in this section has a potential bug or if all paths will end up leading to it just working. There is not a call to `cpuinfo_initialize()` prior to `cpuinfo_has_x86_avx512vnni()`. While I'm here, I could just replace the invocation with `at::cpu::is_cpu_support_avx512_vnni()` which will remove the guarded include. While the logic is backwards, I could also remove the arch check here since the resulting function can now compile and the overall correctness of the function is guarded by the lack of FBGEMM support. Thoughts?